playbooks/aws/openshift-cluster/tasks/launch_instances.yml

```diff
@@ -99,6 +127,8 @@
       env: "{{ env }}"
       host-type: "{{ host_type }}"
       env-host-type: "{{ env_host_type }}"
+      node-type: "{{ node_type }}"
+      env-node-type: "{{ env_node_type }}"
```

Comment:
Same as above, no more env-* tags, that doesn't scale as we only get 10 tags in AWS.